data/campaigns/Under_the_Burning_Suns/scenarios/06b_In_the_Domain_of_Dwarves.cfg

```diff
@@ -696,8 +689,60 @@
                     speaker=Dwarf Sergeant
                     message= _ "Oh, grow a backbone... Huh? Hey, for once the runt was telling the truth. Come on, boys, kill the intruder!"
                 [/message]
+            [/case]
+            [case]
+                value=human
+                [message]
+                    x,y=10,34
+                    message= _ "No, it’s a... human?"
+                [/message]
+                [modify_unit]
+                    [filter]
+                        id=Dwarf Sergeant
+                    [/filter]
+                    facing=se
+                [/modify_unit]
+                [message]
+                    speaker=Dwarf Sergeant
+                    message= _ "What is a human doing here? Come on, boys, kill the intruder!"
+                [/message]
+            [/case]
+            [case]
+                value=monster
+                [message]
+                    x,y=10,34
+                    #po: "it" is referring to the dust devil
+                    message= _ "No, it’s a... what is that?"
+                [/message]
+                [modify_unit]
+                    [filter]
+                        id=Dwarf Sergeant
+                    [/filter]
+                    facing=se
+                [/modify_unit]
+                [message]
+                    speaker=Dwarf Sergeant
+                    #po: "it" is referring to the dust devil
+                    message= _ "What in the Earth’s Guts is that? Kill it!"
```

Comment:
I'm sorry I missed this in the original review - heck, I was the one who merged this - but it doesn't look right to have 'Earth's Guts' capitalised. Are we referring to something specific here? I think these should both be lower-case.

Reply:
I have given it some thought, it also looks to me that both of them should be lower case. At first, I was thinking that at least Earth should be upper case, but probably there is no "Earth" in wesnoth, only "earth". As far as I know, we are referring to the underground, so maybe we should add that in the po hint aswell. How shall we proceed?

Reply:
Capitalised Earth would be referring to our planet, which no-one in Irdya should really know about. (: As you described, earth can refer to the soil/ground/dirt.

If you don't want to go to the trouble of making another PR just for this small change, you can put your revised po hint here and I can update some time tomorrow (I'm about to go to bed soon). Otherwise, just go ahead and make a PR as normal.

I'm thinking something along the lines of:
```
#po: 'earth's guts' is just a reference to the Dust Devil's elemental nature
message= _ "What in the earth’s guts is that? Kill it!"
```

Feel free to make your own suggestions.

Reply:
I still have the branch of this PR, if I were to edit it there, would that be fine? Anyways this is my suggestion:
```
#po: 'earth’s guts' references the underground. What is being talked about is the dust devil.
message= _ "What in the earth’s guts is that? Kill it!"
```

Reply:
I think since this PR is already merged even if you added to the original branch you submitted the request from you'd have to create a new PR anyway.

Thanks, I'll update this in a short while.